rest/adminapitest/resync_test.go

```diff
@@ -220,6 +220,7 @@ func TestResyncRegenerateSequencesPrincipals(t *testing.T) {
 				require.NotEqual(t, originalUserSeq, user.Sequence())
 			}
 
+			collection, ctx = rt.GetSingleTestDatabaseCollection()
```

Comment:
Is this required because taking the db offline w/ persistent config makes the collection reference unusable?

Reply:
yep